docs/_site/build/gh-pages/src/project-displayer.bak

```diff
@@ -0,0 +1,132 @@
+import {LitElement, html} from '@polymer/lit-element';
+import './project-loader.js';
+
+class ProjectDisplayer extends LitElement {
+  static get properties(){
+    return {
+      folder: { type: String },
+
+      clickToLoad: { type: Boolean },
+      forceEmbedLayout: { type: Boolean },
+      view: { type: String },
+      openFile: { type: String },
+      height: { type: Number },
+
+      options: { type: Object },
+      project: { type: Object }
+    };
+  }
+  updateSelectedFile(i){ 
+    var filenames = this.shadowRoot.getElementById('filenames').getElementsByClassName('filename');
+    for (var el = 0; el < filenames.length; el++) {
+      console.log(filenames[el]);
+      filenames[el].className = 'filename';
+    };
+    var filecontents = this.shadowRoot.getElementById('filecontents').getElementsByClassName('filecontents'); 
+    for (var el = 0; el < filecontents.length; el++) {
+      filecontents[el].className = 'filecontents';
+      filecontents[el].style.display = 'none';
+    };
+
+    this.shadowRoot.getElementById('filename-' + i).className += ' selected';
+    this.shadowRoot.getElementById('contents-' + i).className += ' selected';
+    this.shadowRoot.getElementById('contents-' + i).style.display = 'block';
+  }
+
+  constructor(){
+    super();
+    this.project={};
+  }
+  render(){
+    var filenames=[];
```

Comment:
don't use `var` today :-)

Reply:
Also I would add spaces around assignment as well as between () and { - that seems like common style